src/components/DataCard/DataCard.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+
+interface Tag {
+  label: string;
+  color: string;
+}
+
+interface DataCardProps {
+  title: string;
+  tags: Tag[];
+  description: string;
+  publishedAt: string;
+  version: string;
+  sourceUrl: string;
+  downloadUrl: string;
+}
+
+export default function DataCard({
+  title,
+  tags,
+  description,
+  publishedAt,
+  version,
+  sourceUrl,
+  downloadUrl,
+}: DataCardProps) {
+  return (
+    <div className="flex flex-col bg-white border border-gray-200 rounded-md shadow-sm p-6 mb-6 max-w-3xl mx-auto hover:scale-[1.01] transition-transform duration-300">
+      {/* Cabeçalho */}
+      <div className="flex flex-row justify-between items-start mb-4">
+        {/* Bloco do Título e Tags */}
+        <div className="flex flex-col">
+          <h2 className="font-bold text-xl">{title}</h2>
+          <div className="flex flex-wrap gap-2 mt-2">
+            {tags.map((tag) => (
+              <span
+                key={tag.label}
+                style={{
+                  backgroundColor: tag.color,
+                  color: "#fff",
+                }}
+                className="rounded-full px-3 py-1 text-xs font-medium"
+              >
+                {tag.label}
+              </span>
+            ))}
+          </div>
+        </div>
+
+        {/* Bloco dos Botões */}
+        <div className="flex gap-3 flex-shrink-0 ml-4">
+          <a
+            href={sourceUrl}
+            target="_blank"
+            rel="noopener noreferrer"
+            className="flex items-center gap-2 rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-bold text-[#038f39] transition-all hover:bg-green-50 hover:text-green-900 hover:scale-105"
+          >
+            <svg
+              width="16"
+              height="16"
+              viewBox="0 0 24 24"
+              fill="none"
+              stroke="currentColor"
+              strokeWidth="2"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+            >
+              <path d="M18 13v6a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6" />
+              <polyline points="15 3 21 3 21 9" />
+              <line x1="10" y1="14" x2="21" y2="3" />
+            </svg>
```

Comment:
Instead of adding the svg here what we do is add this svg on the `sprite.svg` file (There's a lot of examples there, but any doubts you can ask me on Mattermost).

After adding it to sprite.svg and giving it an id you can use the Icon.tsx component to add it here.